README.md

```diff
@@ -1,51 +1,96 @@
-# nginx_otel
+# NGINX Native OpenTelemetry (OTel) Module
 
-This project provides support for OpenTelemetry distributed tracing in Nginx, offering:
+## What is OpenTelemetry
+OpenTelemetry (OTel) is an observability framework for monitoring, tracing, troubleshooting, and optimizing applications. OTel enables the collection of telemetry data from a deployed application stack.
 
-- Lightweight and high-performance incoming HTTP request tracing
-- [W3C trace context](https://www.w3.org/TR/trace-context/) propagation
-- OTLP/gRPC trace export
-- Fully Dynamic Variable-Based Sampling
+## What is the Native NGINX Otel Module
+The `ngx_otel_module` dynamic module enables NGINX Open-Source or NGINX Plus to send telemetry data to an OTel collector. It provides support for [W3C trace context](https://www.w3.org/TR/trace-context/) propagation, OTLP/gRPC trace exports and offers several benefits over exiting OTel modules, including:
+
+### Better Performance ###
+3rd-party OTel implementations reduce performance of request processing by as much as 50% when tracing is enabled. The NGINX Native module limits this impact to approximately 10-15%.
+
+### Easy Provisioning ###
+Setup and configuration can be done right in NGINX configuration files.
+
+### Fully Dynamic Variable-Based Sampling ###
+The module provides the ability to trace a particular session by cookie/token. [NGINX Plus](https://www.nginx.com/products/nginx/), available as part of a [commercial subscription](https://www.nginx.com/products/), enables dynamic module control via the [NGINX Plus API](http://nginx.org/en/docs/http/ngx_http_api_module.html) and [key-value store](http://nginx.org/en/docs/http/ngx_http_keyval_module.html) modules.
```

Comment:
Both of these use cases are not really obvious without actual NGINX config example. So I'd suggest to keep some generic text here (like "sampling can be controlled by variables, which enables for various use cases, such as ...") and refer to examples section, which would define required configs and links to N+ stuff.

Reply:
Apparently, there is "# or we can trace 10% of user sessions" example below, which shows cookie/token based sampling. So this section can refer to that example, instead.

I don't see a problem with having an example involving commercial features, with clear indication of these, of course (e.g. nginx.org).
The current description still refers to commercial features, but it might be too vague to be useful. How one can enable "dynamic module control" or what does that even mean?

Reply:
Addressed in a subsequent commit, clarifying language and linking to example

https://github.com/nginxinc/nginx-otel/commit/0da0f1537ec73f5683f6ca0d08b1e3a3817577da#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R16